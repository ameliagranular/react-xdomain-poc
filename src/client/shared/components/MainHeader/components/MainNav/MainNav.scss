@import '../../../../../styles/vars';

.MainNav {
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: $navbar_bg_color;
    height: $navbar_height;

    .MainNav_Title {
        font-family: 'Vitesse SSm A', 'Vitesse SSm B', sans-serif;
        font-size: 1rem;
        font-weight: 800;
        color: white;
    }
}
