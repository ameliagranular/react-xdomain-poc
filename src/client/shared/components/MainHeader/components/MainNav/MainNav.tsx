/**
 * The main navigation of the main header of the application.
 */

import * as React from 'react';
import './MainNav.scss';

const MainNav: React.StatelessComponent = () => (
    <div className="MainNav text-center">
        <div className="MainNav_Title">Granular</div>
    </div>
);

export { MainNav };
