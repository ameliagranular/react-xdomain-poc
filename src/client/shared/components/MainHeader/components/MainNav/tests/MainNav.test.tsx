import 'jest'; // necessary for types

import { configure, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import * as Adapter from 'enzyme-adapter-react-16';
import * as React from 'react';

import { MainNav } from '../MainNav';

// Configure Enzyme for use with Jest & ES6+
configure({ adapter: new Adapter() });


const wrapper = (props = {}) => (
    <MainNav {...props} />
);

const doMount = (props = {}) => {
    return mount(wrapper(props));
};

describe('MainNav', () => {
    it('renders expected markup', () => {
        expect(toJson(doMount())).toMatchSnapshot();
    });
});
