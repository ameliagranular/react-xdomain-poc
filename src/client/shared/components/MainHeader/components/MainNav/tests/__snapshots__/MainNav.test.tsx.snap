// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainNav renders expected markup 1`] = `
<MainNav>
  <div
    className="MainNav text-center"
  >
    <div
      className="MainNav_Title"
    >
      Granular
    </div>
  </div>
</MainNav>
`;
